ccdproc/image_collection.py

```diff
@@ -74,23 +72,20 @@ class ImageFileCollection(object):
         easily select subsets of files in the target directory.
         Default is ``None``.
 
-     ext: str or int, optional
-         The extension from which the header and data will be read in all files.
-         Default is ``0``.
+    ext: str or int, optional
```

Comment:
nitpick:  since there is a default value, is it really optional?

Reply:
I think it is optional in the sense that you do not need to include it in the initializer because the default value will work well for most cases.